### PR TITLE
Add Exporter Setup Pipeline

### PR DESCRIPTION
**Pull request by ercl**
This PR implements the Exporter Setup Pipeline, which creates an Exporter and push Controller. This PR is reliant on the Config struct which is in PR #3.

### CODE REVIEW

File: exporters/metric/cortex/cortex.go [hunk 1]
```diff
@@ -3,13 +3,18 @@ package cortex
...
 	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+	"go.opentelemetry.io/otel/sdk/metric/controller/push"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
 )
 
 // Exporter forwards metrics to a Cortex
-type Exporter struct{}
+type Exporter struct {
+	Config
+}
```
Comment (PettitWesley) [inline exporters/metric/cortex/cortex.go:17]: @sonofachamp Is an embedded struct or a member field the best practice in Go for this? (I don't know myself, I probably should stop pretending to be a go expert 🙈)

Comment (connorlindsey) [reply exporters/metric/cortex/cortex.go:17]: This seems pretty common in the other exporters, like this is what the stdout exporter does

```go
type metricExporter struct {
	config Config
}
```

Comment (sonofachamp) [reply exporters/metric/cortex/cortex.go:17]: I'm not sure about best practice, but I don't think it's doing anything other than providing convenience of being able to say `exporter.ConfigField` versus `exporter.config.Field`. I'd suggest being consistent with whatever other exporters in the repository are doing.

Comment (sonofachamp) [reply exporters/metric/cortex/cortex.go:17]: > This seems pretty common in the other exporters, like this is what the stdout exporter does
> 
> ```go
> type metricExporter struct {
> 	config Config
> }
> ```

The nuance is that in this example `Config` is named `config` within the exporter strut whereas in your exporter `Config` is [embedded](https://golang.org/doc/effective_go.html#embedding). It's minor, but I'd advocate for consistency within the repository.

**Pull request merged**